README.md

```diff
@@ -6,41 +6,45 @@
-- View/Create/Delete databases, collections, graphs, stored procedures, documents, and queries
-- Open documents, stored procedures, or queries in the editor
-- Edit documents and persist changes to the cloud
-- Attach database servers to your Workspace using connection strings
+The Azure Databases Explorer provides a comprehensive interface to explore and manage your Azure databases.
+
+- **Discover Database Servers**: Find database servers in your Azure subscription.
+- **Create Database Servers**: Use the `+` button to create a new database server.
+- **Azure Portal Integration**: Open database servers directly in the Azure portal.
+- **Full Database Management**: View, create, and delete databases, collections, graphs, stored procedures, documents, and queries.
+- **Integrated Document Editor**: Open documents, stored procedures, or queries directly in the editor for easy viewing and editing.
+- **Edit and Persist Changes**: Make changes to documents and persist them to the cloud.
+- **Workspace Integration**: Attach database servers to your workspace using connection strings, enabling centralized management.
+- **Flexible MongoDB Connectivity**: Easily establish connections to MongoDB clusters across both Azure and non-Azure environments using a simple connection string.
 
 ![Browse PostgreSQL, CosmosDB, and MongoDB databases](resources/readme/explorer.png)
 
 ## Query Editor
 
-- Write Queries with Syntax Highlighting
-- Show Query history and save/open queries on disk
-- Execute Queries and see the results (currently NoSQL only)
-- Show results in a table, as JSON or in a hierarchical tree view
-- Copy results as JSON or CSV to the clipboard or store them in a file
-- Choose how many results to show at once with paging support
-- Show Query Statistics (consumed RUs, time, size and other metrics)
-- View, Edit and Update single documents from the result view
-- Create new documents with syntax highlighting and partition key validation
+The Query Editor provides a powerful interface to write and execute queries across your databases.
+
+- **Rich Syntax Highlighting**: Write queries with built-in syntax highlighting to improve readability and reduce errors.
+- **Query History and Management**: Save and open queries from disk, and access query history for re-use.
+- **Execute Queries**: Run queries and view the results in multiple formats including Table, JSON, or a hierarchical Tree view (currently NoSQL only).
+- **Flexible Result Management**: Copy query results as JSON or CSV to the clipboard or store them in a file as needed.
+- **Paging Support**: Choose how many results to display at once with convenient paging support, allowing navigation through large datasets.
+- **Query Statistics and Index Advisor**: View detailed query statistics such as consumed RUs, execution time, and index optimization suggestions.
+- **Document Management**: View, edit, and update single documents from the result view directly.

```

Comment:
```suggestion
- **Document Management**: View, edit, and update documents from the result view.

```
The view can open multiple documents (in separate tabs) but this is overall not directly "in" the view.